appveyor.yml

```diff
@@ -77,8 +77,9 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: obvci_install_conda_build_tools.py
     # Workaround for Python 3.4 and x64 bug in latest conda-build.
-    # FIXME: remove this once a new conda-build is released!
-    - cmd: conda install --yes conda-build=1.20.0
+    # FIXME: Remove once there is a release that fixes the upstream issue
+    # ( https://github.com/conda/conda-build/issues/895 ).
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
> ...it *shouldn't* be complex to solve after all

Honestly, I don't have enough understanding of what the problem is here to say. Though I think it has something to do with the workaround added to fix VS2008 64-bit. The right people at least know about the problem at this point. However, I have the impression that one of them is out. So, that may slow the process down.